include/GafferImageUI/ImageGadget.h

```diff
@@ -137,11 +137,11 @@ class GAFFERIMAGEUI_API ImageGadget : public GafferUI::Gadget
 		void setGamma( float gamma );
 		float getGamma() const;
 
-		void setDisplayTransform( GafferImage::ImageProcessorPtr displayTransform );
-		GafferImage::ConstImageProcessorPtr getDisplayTransform() const;
-
-		void setUseGPU( bool useGPU );
-		bool getUseGPU() const;
+		// Use for setting a display color transform that is a Gaffer image processing node that
+		// will run on the CPU.  This approach is deprecated, it is better to set a GPU OCIO
+		// transform on the ViewportGadget that contains the ImageGadget.
+		void setCPUDisplayTransform( GafferImage::ImageProcessorPtr displayTransform );
+		GafferImage::ConstImageProcessorPtr getCPUDisplayTransform() const;
```

Comment:
Can we just rip this out now, along with the internal grade nodes? Hosting nodes inside the Gadget was always [heinous](https://github.com/GafferHQ/gaffer/pull/4767/commits/40bdb950ccbebaf171112a8f5b33791bf630e31e#diff-6008d16dbf200b7352302918f1b983ec7e15c5d3bd3734bdd226d8fb5d04ebdaR918-R928), and now we don't need to translate them to the GPU in the ImageGadget, it seems like we have the opportunity to clean all that up and host them inside the ImageView.

Reply:
Summary of separate discussion : we will do this in a followup PR.